config/map_config.go

```diff
@@ -19,7 +19,8 @@ func NewMap(envVal env.SpaceEnv, flags *Flags) Config {
 	configStr[SpaceStorePath] = "~/.fleek-space"
 	configStr[TextileHubTarget] = "textile-hub-dev.fleek.co:3006"
 	configStr[TextileThreadsTarget] = "textile-hub-dev.fleek.co:3006"
-	configStr[SpaceServicesAPIURL] = "https://td4uiovozc.execute-api.us-west-2.amazonaws.com/dev" // TODO: Get a domain
+	configStr[SpaceServicesAPIURL] = "https://td4uiovozc.execute-api.us-west-2.amazonaws.com/dev"   // TODO: Get a domain
```

Comment:
we should probably pass this values through the build command/flags like the mongo information shown below.
So it is easily configurable on build

Reply:
Yeah this sounds good, maybe for the Textile Hub targets as well.

Reply:
Fixed in 4ef068e